api/models/graphite.go

```diff
@@ -0,0 +1,94 @@
+package models
+
+import (
+	"gopkg.in/raintank/schema.v1"
+	"sort"
+)
+
+type GraphiteRender struct {
+	MaxDataPoints uint32   `json:"maxDataPoints" form:"maxDataPoints"`
+	Targets       []string `json:"target" form:"target" binding:"required"`
+	From          string   `json:"from" form:"from"`
+	Until         string   `json:"until" form:"until"`
+	To            string   `json:"to" form:"to"`
+	Format        string   `json:"format" form:"format"`
```

Comment:
It is not, but it is definitely a query param supported by graphite.  If anything we would return a 400 if the field is anything but "" or "json".